### DISCUSSION TITLE
What's the worst that can happen if you have the voltage switch set wrong?

### DISCUSSION THREAD

**Discussion by megamer29** [sanni/cartreader#370] [Need help]
This may be a dumb question, but I was considering making one of the 7 port adapters, but hearing that the gba/gbc switch affects the full board has me notably worried that I'll accidentally fry something; I think it was mentioned somewhere that if you have it on 3.3v and put in a cartridge that uses 5 or vice versa there might be a short, but I'm not sure what the actual outcome is, and how likely it'd be to potentially permanently damage cartridges or the reader, and my concerns about that are keeping me from committing to the build.

**Comment by sanni** [comment]
The short circuit issue can only happen if you combine the HW4 main PCB in a "forbidden" way.  Basically you are not allowed to attach two single slot adapter PCBs to the HW4 PCB at the same time.

![HW4+5](https://user-images.githubusercontent.com/19169494/169699081-80054a97-81c1-43c5-bef9-374ad7072def.jpg)
white: HW4 main PCB
yellow: single slot PCBs
green: six slot adapter PCB
blue: HW5 main PCB
red: 3d printed stuff

HW5 will remove the battery module and integrated the GBA slot into the main PCB instead. 
HW5 will also allow the 2-slot build option consisting out of the GBA slot and one additional 1-slot adapter PCB.

In all other cases choosing the wrong voltage might result in a bad dump but nothing will short circuit. It's not good for the cartridge but it will not instantly kill it.

**Reply by megamer29** [reply]
Okay, thanks. That makes me feel notably better.
